src/components/button.js

```diff
@@ -1,9 +1,11 @@
-function button(textContent,htmlClass) {
-	if (htmlClass === undefined) {
-		return `<button class="button">${textContent}</button>`;
-	}
+function button(textContent,dataValue,htmlClass) {
+	if (dataValue == undefined) {
```

Comment:
unsells if

Reply:
but if i don't pass the parameter for the data the button will be with a data empty into it, i was thinking doing it like when the class is undefined

Reply:
there's any problem if the button remains with a data-value undefined ? because it'll be like the class when is undefined 
`<button class="button" data-value>${textContent}</button>`

Reply:
ahh I see!

ok then ;)